studybuddies/src/app/services/httperror-interceptor.service.spec.ts

```diff
@@ -0,0 +1,16 @@
+import { TestBed } from '@angular/core/testing';
+
+import { HTTPErrorInterceptorService } from './httperror-interceptor.service';
+
+describe('HTTPErrorInterceptorService', () => {
+  let service: HTTPErrorInterceptorService;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({});
+    service = TestBed.inject(HTTPErrorInterceptorService);
+  });
+
+  it('should be created', () => {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Team-Studybuddies/Studybuddies-Frontend/pullRequest?prid=7202842)
```suggestion
  it("should be created", () => {
```